pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -964,6 +967,103 @@ type MSTeamsConfig struct {
 	HTTPConfig *HTTPConfig `json:"httpConfig,omitempty"`
 }
 
+// RocketChatConfig configures notifications via RocketChat.
+// See https://prometheus.io/docs/alerting/latest/configuration/#rocketchat_config
+type RocketChatConfig struct {
+	// Whether to notify about resolved alerts.
+	// +optional
+	SendResolved *bool `json:"sendResolved,omitempty"`
+	// The API URL for RocketChat.
+	// Defaults to https://open.rocket.chat/ if not specified.
+	// +optional
+	APIURL *string `json:"apiURL,omitempty"`
+	// The channel to send alerts to.
+	// +optional
+	Channel *string `json:"channel,omitempty"`
+	// The sender token. Either Token or TokenFile is required.
+	// +optional
+	Token v1.SecretKeySelector `json:"token,omitempty"`
+	// The file containing the sender token. Mutually exclusive with TokenFile
+	// Either Token or TokenFile is required.
+	// +optional
+	TokenFile *string `json:"tokenFile,omitempty"`
+	// The sender token ID. Mutually exclusive with TokenIDFile.
```

Comment:
Both TokenFile and  TokenID still have the Mutually exclusive text. Would work better if you added as you did for  

```go
	// The file containing the sender token ID. Either TokenID or TokenIDFile is required.
	// +optional
	TokenIDFile *string `json:"tokenIDFile,omitempty"`
```